clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Job.java

```diff
@@ -269,6 +279,39 @@ private void addDisruptionBudget(com.netflix.titus.grpc.protogen.Job grpcJob) {
     }
   }
 
+  private SignedAddressAllocations addSignedAddressAllocations(
+      SignedAddressAllocation signedAddressAllocation) {
+    SignedAddressAllocations grpcSignedAddressAllocations = new SignedAddressAllocations();
+
+    SignedAddressAllocations.AddressLocation grpcAddressLocation =
+        new SignedAddressAllocations.AddressLocation();
+    grpcAddressLocation.setAvailabilityZone(
+        signedAddressAllocation.getAddressAllocation().getAddressLocation().getAvailabilityZone());
+    grpcAddressLocation.setRegion(
+        signedAddressAllocation.getAddressAllocation().getAddressLocation().getRegion());
+    grpcAddressLocation.setSubnetId(
+        signedAddressAllocation.getAddressAllocation().getAddressLocation().getSubnetId());
+
+    SignedAddressAllocations.AddressAllocation grpcAddressAllocation =
+        new SignedAddressAllocations.AddressAllocation();
+    grpcAddressAllocation.setAddress(signedAddressAllocation.getAddressAllocation().getAddress());
+    grpcAddressAllocation.setUuid(signedAddressAllocation.getAddressAllocation().getUuid());
+    grpcAddressAllocation.setAddressLocation(grpcAddressLocation);
+
+    grpcSignedAddressAllocations.setAddressAllocation(grpcAddressAllocation);
+    grpcSignedAddressAllocations.setAuthoritativePublicKey(
+        signedAddressAllocation.getAuthoritativePublicKey().toStringUtf8());
+    grpcSignedAddressAllocations.setHostPublicKey(
+        signedAddressAllocation.getHostPublicKey().toStringUtf8());
+    grpcSignedAddressAllocations.setHostPublicKeySignature(
+        signedAddressAllocation.getHostPublicKeySignature().toStringUtf8());
+    grpcSignedAddressAllocations.setMessage(signedAddressAllocation.getMessage().toStringUtf8());
+    grpcSignedAddressAllocations.setMessageSignature(
+        signedAddressAllocation.getMessageSignature().toStringUtf8());
+
+    return grpcSignedAddressAllocations;
+  }
```

Comment:
What is the method supposed to do?

Reply:
Looks like `buildSignedAddressAllocation` is duplicate functionality(like @jeyrschabu said)?

Reply:
this is used to construct the titus internal model SignedAddressAllocations from the grpc object